src/components/MultiHopTrade/components/FeeModal/FeeBreakdown.tsx

```diff
@@ -24,13 +24,17 @@ const AmountOrFree = ({ isFree, amountFiat }: { isFree: boolean; amountFiat: str
 
 export const FeeBreakdown = () => {
   const translate = useTranslate()
-  const votingPower = useAppSelector(selectVotingPower)
+  const votingPowerParams: { feeModel: ParameterModel } = useMemo(
+    () => ({ feeModel: 'swapper' }),
+    [],
+  )
+  const votingPower = useAppSelector(state => selectVotingPower(state, votingPowerParams))
   const sellAmountUsd = useAppSelector(selectInputSellAmountUsd)
   const { foxDiscountUsd, foxDiscountPercent, feeUsdBeforeDiscount, feeBpsBeforeDiscount } =
     calculateFees({
       tradeAmountUsd: bnOrZero(sellAmountUsd),
       foxHeld: votingPower !== undefined ? bn(votingPower) : undefined,
-      parameters: feeCurveParameters.swapper,
+      feeModel: 'swapper',
```

Comment:
I personally am fine with and like explicit string unions, but I can hear "use an enum" in my head based on past reviews haha

Reply:
agreed!